js/cfi_interpreter.js

```diff
@@ -55,7 +55,6 @@ var obj = {
     //   the reading system, as it stands now.
     getContentDocHref : function (CFI, packageDocument, classBlacklist, elementBlacklist, idBlacklist) {
 
-        var $packageDocument = $(packageDocument);
         var decodedCFI = decodeURI(CFI);
```

Comment:
The `packageDocument` function parameter is not a jQuery object anymore in `searchLocalPathForHref`, `getContentDocHref `, and `retrieveItemRefHref `. This is good, but this is also a major API change, which could break existing consumer / client code. Any thoughts?

Reply:
Follow-up: I suppose $(obj) === obj when obj is already a $ jQuery object, right? So this "normalisation" step allows for both vanilla JS objects (DOM éléments) and jQuery objects to be used as function parameter ... so I retract my comment.